src/ColumnSchema.php

```diff
@@ -33,9 +33,9 @@ public function defaultPhpTypecast($value)
         return $this->phpTypecast($value);
     }
 
-    public function dbTypecast($value)
+    public function dbTypecast(mixed $value): mixed
     {
-        if ($this->getType() === SchemaPDOMssql::TYPE_BINARY && $this->getDbType() === 'varbinary') {
+        if ($this->getType() === Schema::TYPE_BINARY && $this->getDbType() === 'varbinary') {
```

Comment:
How about refactoring in next PR - SchemaPDOMssql::DB_TYPE_VARBINARY = 'varbinary' (this and in other places)?

Reply:
+1